user-guide/Standard_Apps/Monitoring_Solutions/Performance_Analyzer/Performance_Analyzer_Library.md

```diff
@@ -0,0 +1,428 @@
+---
+uid: Performance_Analyzer_Library
+---
+
+# Problem
```

Comment:
Similar to the comment above: please add a clear title and introduction to make it clear what the page is about.

Reply:
Done